torch/linalg/__init__.py

```diff
@@ -1676,20 +1743,21 @@
 
 .. note:: This function uses LAPACK and MAGMA for CPU and CUDA inputs respectively.
 
-.. warning:: The QR decomposition is only locally unique when the input matrix has
-             independent columns. If this is not the case, different platforms, like NumPy,
+.. warning:: The QR decomposition is only unique up to sign of the diagonal of `R` when the
+             input matrix has independent columns.
+             If this is not the case, different platforms, like NumPy,
              or inputs on different devices, may produce different valid decompositions.
 
 .. warning:: Backpropagation is only supported if the first `k = min(m, n)` columns
```

Comment:
"Backpropagation is only supported if..." -> "Gradient computation using ??? is only supported if..."